src/Athena/Content/js/site.js

```diff
@@ -0,0 +1,5 @@
+﻿const jquery = require("jquery");
+window.$ = window.jQuery = jquery;
+
```

Comment:
You will need to `require('collapsible.js`);` here if you want it included in the bundle